### ISSUE TITLE
Rails app fails to boot when installing 'split'

### ISSUE THREAD

**Issue by dhruvparmar372** [splitrb/split#601]
**Describe the bug**
Rails app boot fails with following error

```
NoMethodError: undefined method `helper' for ActionController::API:Class
```
Reverting the change made here https://github.com/splitrb/split/pull/586/files seems to fix the issue. 

**To Reproduce**
1. Install 'split'
2. Boot rails app

**Expected behavior**
App should boot

**Additional context**
Rails version 6.0.1.rc1

**Comment by andrehjr** [comment]
Hi @dhruvparmar372! Thanks for filing this issue.

I'll release 3.4.1 with a fix. This is breaking Rails API Controllers.

**Comment by andrehjr** [comment]
I released 3.4.1 a few minutes ago, can you check if everything looks good on your end @dhruvparmar372 ? I tested with a few projects over here, and it boots just fine right now.

**Comment by dhruvparmar372** [comment]
Yep `3.4.1` works well! Thank you for the quick patch 🙌
